content/docs/0.8.x/continuous_delivery/multi_stage/index.md

```diff
@@ -35,16 +35,68 @@ spec:
     - name: "production"
 ```
 
-### Approval Strategy
+### Definition of Sequence in a Stage
+
+After defining the stages, sequences can be added to a stage. A sequence is an ordered list of tasks that are triggered sequentially. A `sequence` has the properties:
 
-The approval strategy specifies the kind of approval, which is required before deploying an artifact into the next stage. The approval strategy can be defined based on the evaluation result `pass` and `warning`. Keptn supports the following approval strategies for the evaluation results `pass` and `warning`:
+* `name`: A unique name of the sequence
+* `on` (optional): An array of events that trigger the sequence.
+* `tasks`: An array of tasks executed by the sequence in the declared order.
+
+**Example:** Extended shipyard with a delivery sequence in all three stage:
+
+```yaml
+apiVersion: spec.keptn.sh/0.2.0
+kind: "Shipyard"
+metadata:
+  name: "shipyard-sockshop"
+spec:
+  stages:
+    - name: "dev"
+      sequences:
+      - name: "delivery"
+        tasks: 
+        - name: "deployment"
+        - name: "release"
+    - name: "hardening"
+      sequences:
+      - name: "delivery"
+        triggeredOn:
+          - event: "dev.delivery.finished"
+        tasks: 
+        - name: "deployment"
+        - name: "release"
+    - name: "production"
+      sequences:
+      - name: "delivery"
+        triggeredOn:
+          - event: "hardening.delivery.finished"
+        tasks: 
+        - name: "deployment"
+        - name: "release"
+```
+
+### Reserved Keptn Tasks
+
+Keptn supports a set of opiniated tasks for declaring a delivery or remediation sequence: 
+
+* approval
+* deployment
+* evaluation
+* release
+* remediation
+* rollback (missing)
+* test
+
+#### Approval
+
+The approval task stops the sequence for an approval, which is required before deploying an artifact into the next stage. The approval strategy can be defined based on the evaluation result `pass` and `warning`. Keptn supports the following approval strategies for the evaluation results `pass` and `warning`:
```

Comment:
Consider rephrasing the first sentence to
"The approval task intercepts the sequence and waits for a required approval (e.g., before deploying an artifact into the next stage)"

Reply:
very good point. Now it is more generic. thx